### PR TITLE
Add sha256sum to dbschema.rb

### PR DESCRIPTION
**Pull request by cbcoutinho**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `dbschema` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/d/dbschema.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask "dbschema" do
   version "9.6.1"
-  sha256 :no_check # required as upstream package is updated in-place
+  sha256 "37b8ef2fa348a51bdf0cadd2fdfdd7a3c33e4eee81c4f60c16fe0f0e2a246f39" # See sha256sums at https://dbschema.com/download/sha256sums
```
Comment (p-linnane) [inline Casks/d/dbschema.rb:3]:
```suggestion
  sha256 "37b8ef2fa348a51bdf0cadd2fdfdd7a3c33e4eee81c4f60c16fe0f0e2a246f39"
```

Comment (p-linnane) [reply Casks/d/dbschema.rb:3]: Can you squash your commits after applying this suggestion? Thanks!

**Comment by bevanjkay** [comment]
The vendor has a long history of updating the package without changing the version number, which is why we cannot use the checksum.
See https://github.com/Homebrew/homebrew-cask/pull/118464 for some context.

**Comment by p-linnane** [comment]
Closing per above.

**Pull request closed without merging**